Meadowlark-js/packages/meadowlark-authz-server/src/plugin/AuthorizationPluginLoader.ts

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import R from 'ramda';
+import { Logger, writeErrorToLog } from '@edfi/meadowlark-core';
+import { AuthorizationStoreInitializer } from './AuthorizationStoreInitializer';
+import { AuthorizationStorePlugin } from './AuthorizationStorePlugin';
+import { NoAuthorizationStorePlugin } from './NoAuthorizationStorePlugin';
+
+let loadedAuthorizationStore: AuthorizationStorePlugin = NoAuthorizationStorePlugin;
+
+export async function loadAuthorizationStore() {
+  if (loadedAuthorizationStore !== NoAuthorizationStorePlugin) return;
+
+  if (process.env.AUTHORIZATION_STORE_PLUGIN == null) return;
```

Comment:
Authorization servers have their own plugin store definition to allow for separation, though at least initially our existing backends will provide it.

Reply:
So this is the loading point for any authorization integration, correct? So when we do an OAuth provider, or if we do an Auth0 integration, they would be and Authorization Store Plugin, and loaded here.

Reply:
No, this is just the integration point for Meadowlark's Authorization server's "authorization store", meaning its database. We're only implementing MongoDB for now, but this is where an alternate database implementation (PostgreSQL, etc.) would plug in.